import/import.go

```diff
@@ -0,0 +1,594 @@
+
+	// connect to v1beta1 client
+	ctx := context.Background()
+	platformClient, err := platform.NewPlatformClient(host, token, "import")
+	if err != nil {
+		log.Fatalf("Failed to create platform client: %v", err)
+	}
+
+	iamClient, err := iam.NewIamClient(host, token, "import")
+	if err != nil {
+		log.Fatalf("Failed to create iam client: %v", err)
+		return
+	}
+
+	// set terraform provider configuration
+	var importString string
+	importString += fmt.Sprintf(`
+	terraform {
+		required_providers {
+			astro = {
+				source = "registry.terraform.io/astronomer/astro"
+			}
+		}
+	}
+	
+	provider "astro" {
+		organization_id = "%s"
+		host = "%s"
+		token = "%s"
+	}
+
+	`, organizationId, host, token)
+
+	//	for each resource, we get the list of entities and generate the terraform import command
+
+	resourceHandlers := map[string]func(context.Context, *platform.ClientWithResponses, *iam.ClientWithResponses, string) string{
+		"workspace":  handleWorkspaces,
+		"deployment": handleDeployments,
+		"cluster":    handleClusters,
+		"api_token":  handleApiTokens,
+		"team":       handleTeams,
+		"team_roles": handleTeamRoles,
+		"user_roles": handleUserRoles,
+	}
+
+	for _, resource := range resources {
+		handler, exists := resourceHandlers[resource]
+		if !exists {
+			log.Println("Resource not supported: ", resource)
+			continue
+		}
+		importString += handler(ctx, platformClient, iamClient, organizationId)
+	}
+
+	// write the terraform configuration to a file
+	err = os.WriteFile("import.tf", []byte(importString), 0644)
+	if err != nil {
+		log.Fatalf("Failed to write import configuration to file: %v", err)
+		return
+	}
+
+	log.Println("Successfully wrote import configuration to import.tf")
+
+	// Generate the corresponding terraform HCL configuration for each import block
+	err = runTerraformCommand()
+	if err != nil {
+		log.Fatalf("Failed to run Terraform command: %v", err)
+		return
+	}
+
+	log.Println("Terraform plan executed successfully")
+}
+
+func runTerraformCommand() error {
+	// delete the generated.tf file if it exists
+	filename := "generated.tf"
+	// Check if the file exists
+	if _, err := os.Stat(filename); err == nil {
+		// File exists, so delete it
+		err = os.Remove(filename)
+		if err != nil {
+			return err
+		}
+		log.Printf("Successfully deleted %s", filename)
+	} else if os.IsNotExist(err) {
+		// File does not exist, nothing to do
+		log.Printf("%s does not exist, no need to delete", filename)
+	} else {
+		// Some other error occurred
+		return err
+	}
+
+	// terraform plan to generate the configuration
+	cmd := exec.Command("terraform", "plan", "-generate-config-out=generated.tf")
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+
+	if err := cmd.Run(); err != nil {
+		return err
+	}
+
+	// terraform apply to import the resources
+	cmd = exec.Command("terraform", "apply", "-auto-approve")
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+
+	if err := cmd.Run(); err != nil {
+		return err
+	}
```

Comment:
currently, script is set to automatically run `terraform apply` on the resources - do we want to keep it this way? or give users more control and let them decide what they want to import?

Reply:
we should not apply it
just give them the file